store/tikv/mock-tikv/mvcc.go

```diff
@@ -300,6 +434,12 @@ type RawKV interface {
 	RawDelete(key []byte)
 }
 
+// MVCCDebugger is for debugging.
+type MVCCDebugger interface {
```

Comment:
A interface should be simple...The more methods a interface contain, the harder to implement the interface.
So I remove some non-essential methods from `MVCCStore` to `MVCCDebugger`, those methods are only necessary for debugging purpose.